src/context.cc

```diff
@@ -644,7 +648,7 @@ void Context::updateUI(const UIElements &what) {
 
         if (what.display_time_entries && user_) {
             if (what.open_time_entry_list) {
-                time_entry_editor_guid_ = "";
+                UI()->resetTimeEntryGUID();
```

Comment:
@IndrekV It's my hot-fix https://github.com/toggl/toggldesktop/commit/9ca209f4b493fee8ea76e3b68200591a2138bd8c for this issue (https://github.com/toggl/toggldesktop/pull/3033#issuecomment-502993150)

The original code forgot to reset the TimeEntryEditorGUID, so it cases the Editor doesn't appear if we select same Time Entry